ludwig/hyperopt/execution.py

```diff
@@ -368,7 +368,14 @@ def _get_best_model_path(self, trial_path, analysis):
             sync_client.sync_down(remote_checkpoint_dir, trial_path)
             sync_client.wait()
         self._remove_partial_checkpoints(trial_path)  # needed by get_best_checkpoint
-        return analysis.get_best_checkpoint(trial_path.rstrip("/"))
+        mod_path = None
+        try:
+            mod_path = analysis.get_best_checkpoint(trial_path.rstrip("/"))
+        except Exception:
+            logger.warning(
+                "Cannot get best model path for {trial_path} due to exception below:\n" f"{traceback.format_exc()}"
```

Comment:
nit: move the `f` to the beginning:

`f"Cannot get best model path for {trial_path} due to exception below:\n{traceback.format_exc()}"`

Reply:
Done